packages/cli/src/commands/sandbox/logs.ts

```diff
@@ -153,7 +153,7 @@ export const logsCommand = new commander.Command('logs')
           console.log(`\nLogs for sandbox ${asBold(sandboxID)}:`)
         }
 
-        const isRunningPromise = listSandboxes()
+        const isRunningPromise = listSandboxes({ state: ['running'] })
```

Comment:
I think we can check this more efficiently now, you can do sandbox get, right?

Reply:
`Sandbox.getInfo` is not in the beta branch